src/BufferCopy/ColorBufferToRDRAM_BufferStorageExt.h

```diff
@@ -0,0 +1,30 @@
+#include "ColorBufferToRDRAM.h"
+
+#include <Textures.h>
+#include <FBOTextureFormats.h>
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+#include "inc/ARB_buffer_storage.h"
+
+
+class ColorBufferToRDRAM_BufferStorageExt : public ColorBufferToRDRAM
+{
+public:
+	ColorBufferToRDRAM_BufferStorageExt();
+	~ColorBufferToRDRAM_BufferStorageExt() = default;
+
+private:
+	void _init() override;
+	void _destroy() override;
+	bool _readPixels(GLint _x0, GLint _y0, GLsizei _width, GLsizei _height, u32 _size, bool _sync)  override;
+	void _cleanUp()  override;
+	void _initBuffers(void) override;
+	static const int _numPBO = 10;
```

Comment:
Yes, 10 is overkill. Besides it will be bad to have lag in 9 frames. I think 2 should be used there and in ColorBufferToRDRAM_GL